scripts/30_prepare_disk.sh

```diff
@@ -208,6 +208,51 @@ partition_resize()
     sfdisk --quiet "${TARGET_STORAGE_DEVICE}" < "${SYSTEM_UPDATE_DIR}/${PARTITION_TABLE_FILE}"
 }
 
+backup_data()
+{
+    temp_mount_dir=$(mktemp -d)
+
+    sfdisk --quiet --dump "${TARGET_STORAGE_DEVICE}" | \
+    while IFS="${IFS}:=," read -r disk_device _ disk_start _ disk_size _; do
+        if grep -q "${disk_device}" /proc/mounts; then
```

Comment:
Shellcheck won't complain, but i but it is more consistent to also add quotes to "/proc/mount", same goes for the restore.